ban/http/resources.py

```diff
@@ -296,6 +296,23 @@ class Group(WithHousenumbers):
 class Postcode(WithHousenumbers):
     model = models.PostCode
     order_by = [model.code, model.municipality]
+    param_whitelist = ['code']
+
+    def get_where_clause(self, req, qs):
+        if hasattr(self, 'param_whitelist'):
+            for param in self.param_whitelist:
+                if req.get_param_as_list(param):
```

Comment:
Put it in a variable only once (no need to call `get_param_as_list` twice), and then check for its emptyness.